channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudContextBuilder.java

```diff
@@ -91,13 +52,18 @@ public JiraCloudContext build(ConfigurationModel globalConfig, DistributionJobMo
         DistributionJobDetailsModel distributionJobDetails = jobModel.getDistributionJobDetails();
         JiraCloudJobDetailsModel jiraCouldJobDetails = distributionJobDetails.getAsJiraCouldJobDetails();
 
-        IssueConfig issueConfig = new IssueConfig();
-        issueConfig.setProjectName(jiraCouldJobDetails.getProjectNameOrKey());
-        issueConfig.setIssueCreator(jiraCouldJobDetails.getIssueCreatorEmail());
-        issueConfig.setIssueType(jiraCouldJobDetails.getIssueType());
-        issueConfig.setCommentOnIssues(jiraCouldJobDetails.isAddComments());
-        issueConfig.setResolveTransition(jiraCouldJobDetails.getResolveTransition());
-        issueConfig.setOpenTransition(jiraCouldJobDetails.getReopenTransition());
+        // FIXME add custom fields
+        JiraIssueConfig issueConfig = new JiraIssueConfig(
+            jiraCouldJobDetails.getProjectNameOrKey(),
+            jiraCouldJobDetails.getProjectNameOrKey(),
```

Comment:
Are we passing in the projectNameOrKey twice on purpose? or was this an accident?

Reply:
I'm not entirely sure we use all of these fields and it's something that needs a closer look, but this object has "project name" and "project key" fields and I'm passing it into both just in case. In Jira APIs you can use both interchangeably (unlike the project id).